doc/install/mtls-configuration.md

```diff
@@ -12,137 +12,42 @@ At the time of writing there is [an RFC](https://github.com/Kuadrant/architectur
 
 ## Prerequisites
 
+- Have [Istio](https://istio.io/) as the Gateway API provider installed.
 - You have installed Kuadrant in a Kubernetes cluster.
 - Additionally, you have at least 1 AuthPolicy or RateLimitPolicy attached to your Gateway or HTTPRoute.
```

Comment:
hmm this is unfortunate perhaps we should look at an enhancement here to allow MTLS without needing an additional policy in place?

Reply:
If there are no policies, then there's no kuadrant data plane components to ensure mtls between?

Reply:
let me add a note to make it very clear

Reply:
Ack.
Sounds better to *not* have this requirement called out, provided the implementation will kick in when needed even if mtls was configured before a policy existed (which I believe it should)

Reply:
pushed a note. Let me know what you think.

> provided the implementation will kick in when needed even if mtls was configured before a policy existed (which I believe it should)

Currently not implemented. I captured this feature in https://github.com/Kuadrant/kuadrant-operator/issues/1284 and included as child of the parent issue: mTLS Configuration #1152